contracts/cairo/src/escrow.cairo

```diff
@@ -282,4 +282,43 @@ mod Escrow {
 
         self.emit(Withdraw { order_id, address: self.mm_starknet_wallet.read(), amount });
     }
+
+    #[l1_handler]
+    fn withdraw_batch(
+        ref self: ContractState,
+        from_address: felt252,
+        order_ids: Array<u256>,
+        recipient_addresses: Array<EthAddress>,
+        amounts: Array<u256>
+    ) {
+        let eth_transfer_contract_felt: felt252 = self.eth_transfer_contract.read().into();
+        assert(eth_transfer_contract_felt == from_address, 'Only ETH_TRANSFER_CONTRACT');
```

Comment:
```suggestion
        assert(from_address == eth_transfer_contract_felt, 'Only YAB_TRANSFER_CONTRACT');
```
To replicate the same change made in withdraw function in an open PR (for better understandability)